docs/docs/getting-started/structure.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 2
+title: Project Structure
+---
+
+### Directories
+
+```
+- /
+ - /${language} # The language of the kernel (python, R, rust, etc)
+    - /${identifier} # Kernel "type" (base, base-gpu, databricks, spark, etc)
+        - /$(version) # If applicable, the version of the language being used
+ - /docs # Docusaurus site (you're here!)
+```
+
+For some of the existing kernels, this looks like
+```
+- /
+  - /R/base/4.3.0
+  - /python
+    - /base
+      - /3.9
+      - /3.10
+```
+
+If we were to add a new language, such as Rust @ `1.70.0`:
+```
+- /
+  - rust/base/1.70.0 # Add a new rust/base directory, then seed the initial version
+```
+
+To add an additional Python version, such as `3.11`:
+```
+- /
+  - /python
+    - /base
+      - /3.9
+      - /3.10
+      - /3.11 # We just add a new subdirectory
+```
+
+Note, that when adding kernel directories, `/docker-bake.hcl` will also need to be
```

Comment:
and how do we do that?

Reply:
There's an assumption that people who are adding wholly new kernel directories, have a base understanding of the syntax and whatnot. If they don't, there's comments as well as links to reference documents, inside the `docker-bake.hcl` file as well

Reply:
You should point out that assumption and be explicit in the documentation about that